docs/k0s-install.md

```diff
@@ -0,0 +1,306 @@
+  -l, --logging stringToString   Logging Levels for the different components (default [konnectivity-server=1,kube-apiserver=1,kube-controller-manager=1,kube-scheduler=1,kubelet=1,kube-proxy=1,etcd=info,containerd=info])
+      --profile string           worker profile to use on the node (default "default")
+      --token-file string        Path to the file containing join-token.
+
+Global Flags:
+      --data-dir string        Data Directory for k0s (default: /var/lib/k0s). DO NOT CHANGE for an existing setup, things will break!
+      --debugListenOn string   Http listenOn for debug pprof handler (default ":6060")
+```
+
+For example runnig a single node k0s cluster on Ubuntu 20.10:
+
+```
+$ k0s install controller --enable-worker
+INFO[2021-02-24 11:05:42] no config file given, using defaults         
+INFO[2021-02-24 11:05:42] creating user: etcd                          
+INFO[2021-02-24 11:05:42] creating user: kube-apiserver                
+INFO[2021-02-24 11:05:42] creating user: konnectivity-server           
+INFO[2021-02-24 11:05:42] creating user: kube-scheduler                
+INFO[2021-02-24 11:05:42] Installing k0s service
+```
+
+Run k0s as a service
+
+```
+$ systemctl start k0scontroller.service
+```
+
+Check service status:
+
+```
+# systemctl status
+● k0s
+    State: running
+     Jobs: 0 queued
+   Failed: 0 units
+    Since: Wed 2021-02-24 10:49:41 UTC; 30min ago
+   CGroup: /
+   ...
+```
+
+Poll cluster for the status:
+
+```
+$ k0s status
+Version: v0.11.0-beta.2-16-g02cddab
+Process ID: 9322
+Parent Process ID: 1
+Role: controller+worker
+Init System: linux-systemd
+```
+
+Fetch nodes
+```
+$ k0s kubectl get nodes
+NAME   STATUS   ROLES    AGE    VERSION
+k0s    Ready    <none>   4m6s   v1.20.4-k0s1
+```
+
+## Under the hood
+
+The command will create service stub file which can be found at:
+
+```
+$ cat /etc/systemd/system/k0scontroller.service 
+[Unit]
+Description=k0s - Zero Friction Kubernetes
+Documentation=https://docs.k0sproject.io
+ConditionFileIsExecutable=/root/go/src/github.com/k0sproject/k0s/k0s
+
+After=network-online.target 
+Wants=network-online.target 
+
+[Service]
+StartLimitInterval=5
+StartLimitBurst=10
+ExecStart=/root/go/src/github.com/k0sproject/k0s/k0s controller --enable-worker=true
+
+RestartSec=120
+Delegate=yes
+KillMode=process
+LimitCORE=infinity
+TasksMax=infinity
+TimeoutStartSec=0
+LimitNOFILE=999999
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+```
+The command will create users which can be listed:
+```
+$ cat /etc/passwd
+...
+etcd:x:997:996::/var/lib/k0s:/usr/sbin/nologin
+kube-apiserver:x:996:995::/var/lib/k0s:/usr/sbin/nologin
+konnectivity-server:x:995:994::/var/lib/k0s:/usr/sbin/nologin
+kube-scheduler:x:994:993::/var/lib/k0s:/usr/sbin/nologin
+```
+
+
+
+## k0sctl
+
+k0sctl is a command line tool for bootstrapping and managment of k0s clusters. k0sctl Install instructions can be found [here](https://github.com/k0sproject/k0sctl#installation)
```

Comment:
```suggestion
k0sctl is a command-line tool for bootstrapping and the management of k0s clusters. Installation instructions can be found in the [k0sctl github repository](https://github.com/k0sproject/k0sctl#installation).
```

It's usually good to avoid link texts like "here". The hyperlinks are "metadata", the content should make sense without the links being clickable.

Command-line is usually hyphenated.